src/components/policy/policy_regular/src/status.cc

```diff
@@ -57,6 +57,9 @@ policy::UpdateNeededStatus::UpdateNeededStatus()
 void policy::UpdateNeededStatus::ProcessEvent(
     policy::UpdateStatusManagerInterface* manager, policy::UpdateEvent event) {
   switch (event) {
+    case kUpdateForNextInQueue:
+      manager->SetNextStatus(std::make_shared<UpdateNeededStatus>());
```

Comment:
Same here, seems redundant since we are already in this state.

Reply:
@jacobkeeler deleted in [ca790d7](https://github.com/smartdevicelink/sdl_core/pull/3208/commits/ca790d7a1ebe90caa163b4fcf951f512cdaddcec)